### ISSUE TITLE
Public exposure of root / DatabaseStatus web pages

### ISSUE THREAD

**Issue by jeremy303** [FraunhoferIOSB/FROST-Server#93]
I would think we shouldn't publicly expose the 'DatabaseStatus' page (and perhaps the root page) that FROST serves up.  We can handle this via our reverse proxy but are their scenarios where it would be valuable to the operations team to have access to this page?  Is there a different recommended approach?

Thanks!

**Comment by mjacoby** [comment]
I agree that it is not a good idea to make the "update database" functionality publicly accessible. However, for administrators and developers this might be useful information/function. As for now, FROST-Server does not provide any means to limit access to those pages. It needs to be managed via the web server/reverse proxy configuration upon deployment as you are doing now.

If you have any proposal how to address this issue (independent of the way access control is realized) feel free to post it here to discuss it. Even better, if you already have a solution you can add a pull request.

**Comment by HolySamosa** [comment]
Thanks, Michael.

One small suggestion would be to remove the "Database Status and Update" link from the main page. The rest of the main page content is fine, even desirable for a public audience.  This way we could protect DatabaseStatus via proxy/server config, leaving the main page free to be served without admin-only links.

Perhaps DatabaseStatus could be hung off a new `/admin` route, which would allow for future growth of admin features, be easy to secure via proxy / server config and easy to remember.

**Comment by AzazelSensei** [comment]
Opened https://github.com/FraunhoferIOSB/FROST-Server/pull/2419 — docs for protecting DatabaseStatus when auth is off, and the admin role when it is on.
